registry/proxy/proxyblobstore_test.go

```diff
@@ -189,13 +190,13 @@ func makeTestEnv(t *testing.T, name string) *testEnv {
 func makeBlob(size int) []byte {
 	blob := make([]byte, size)
 	for i := 0; i < size; i++ {
-		blob[i] = byte('A' + rand.Int()%48)
+		blob[i] = byte('A' + randSource.Int()%48)
 	}
 	return blob
 }
 
 func init() {
-	rand.Seed(42)
+	randSource = *rand.New(rand.NewSource(42))
 }
```

Comment:
**suggestion:** It'd be nice if we could move this `init` to the top of this file. I know we have a bunch of cases like this all over this repo where the `init` is hidden in dark corners from the eyes of the user, but it'd be nice to make this easier to spot for whoever opens this file.

Reply:
Separate pr?

Reply:
That works 👍